app/services/affiliate-slots-units.json

```diff
@@ -211,5 +211,26 @@
     },
     "country": ["US", "CA", "NL", "AU", "NZ"],
     "isBig": true
+  },
+  {
+    "campaign": "ddb",
+    "category": "ddb",
+    "tagline": "Related Posts",
+    "image": "https://static.wikia.nocookie.net/0cebb500-f8c3-4f44-b4aa-f44e33fc2319",
+    "heading": "The Official Digital Toolset for Dungeons & Dragons",
+    "subheading": "Visit D&D Beyond",
+    "launchOn": "2019-12-12T00:00:00Z",
```

Comment:
Is that correct? It's 12/11 4pm PDT.

Reply:
No. Definitely 12/12. Will get a specific time.

Reply:
Going with 11am (PST) on 2019-12-12